docs/habana-gaudi.md

```diff
@@ -142,11 +142,12 @@ The [`SFTTrainer`](https://huggingface.co/docs/trl/sft_trainer) from `trl` does
 
 ## Install and run InstructLab with Intel Gaudi
 
-Install `InstructLab` from checkout with additional dependency `habana`:
+Install `InstructLab` from checkout with additional dependencies:
 
 ```shell
 . $HABANALABS_VIRTUAL_DIR/bin/activate
-pip install './instructlab[habana]'
+pip install -r instructlab/requirements.txt -r instructlab/requirements-hpu.txt
+pip install ./instructlab
```

Comment:
can't this be a one-liner?

Reply:
yes, I changed the text. Thanks for the suggestion!